src/generate/util.ts

```diff
@@ -44,15 +45,41 @@ export const template = async (
   }
 };
 
+const updateDependencies = async (dependencies: Record<string, string>) => {
+  const promises = Object.entries(dependencies).map(async ([name, version]) => {
+    if (version === "*") {
```

Comment:
Do we want to encapsulate `"*"` in a constant that can be imported in the different command impls?

Reply:
I considered it. I think this is good enough for now but I do want to see if I can find a better sentinel value that is something other than a string to convey this better.